packages/plugin-flex/src/clients/ServerlessClient.ts

```diff
@@ -156,6 +152,42 @@ export default class ServerlessClient {
     return build;
   }
 
+  /**
+   * Deletes the {@link EnvironmentInstance}
+   * @param serviceSid   the service sid
+   * @param environment  the environment sid
+   */
+  async deleteEnvironment(serviceSid: string, environmentSid: string): Promise<boolean> {
+    return this.client
+      .get(serviceSid)
+      .fetch()
+      .then(async (service) => service.environments().get(environmentSid).remove());
+  }
+
+  /**
+   * Returns the {@link EnvironmentInstance}
+   * @param serviceSid  the service sid
+   * @param pluginName  the plugin name
+   */
+  async getEnvironment(serviceSid: string, pluginName: string): Promise<EnvironmentInstance> {
+    return this.client
+      .get(serviceSid)
+      .fetch()
+      .then(async (service) => {
+        if (!service) {
+          return Promise.resolve({} as EnvironmentInstance);
```

Comment:
You should return `null` - not an empty object.

Reply:
i actually had that originally but then changed it bc of a similar function below; i can change it back